### DISCUSSION TITLE
Welcome to abstract-thinking Discussions!

### DISCUSSION THREAD

**Discussion by spamegg1** [spamegg1/abstract-thinking#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mohammadazeemwani** [comment]
Wow;;
a whole new world
      
^^ will you work on this?

**Reply by spamegg1** [reply]
Yeah I'm working on it now, but it's a challenging problem, it will take time! Very difficult to explain.

**Reply by mohammadazeemwani** [reply]
Ok then. 
Good Luck
